src/sentry/migrations/0167_rm_copied_useroptions.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 1.11.29 on 2021-03-01 20:08
+
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def delete_duplicate_useroption_rows(apps, schema_editor):
+    """
+    Delete the rows in UserOption that have already been copied over
+    to the NotificationSetting table
+    """
+    UserOption = apps.get_model("sentry", "UserOption")
+
+    for user_option in RangeQuerySetWrapperWithProgressBar(UserOption.objects.all()):
+        if (
+            user_option.key == "workflow:notifications"
+            or user_option.key == "mail:alert"
+            or user_option.key == "deploy-emails"
```

Comment:
Nit: `user_option.key in (<values>)`

Reply:
I think this migration would also be a good time to clean up the other unused keys. For example: https://github.com/getsentry/sentry/pull/5318 and anything else marked "unused" in this list:
https://github.com/getsentry/sentry/blob/master/src/sentry/models/useroption.py#L109.